tests/dags/test_issue_1225.py

```diff
@@ -43,7 +43,7 @@ def fail():
 dag1_task1 = DummyOperator(
     task_id='test_backfill_pooled_task',
     dag=dag1,
-    pool='test_backfill_pooled_task_pool',
+    pool='default_pool',
```

Comment:
I don't think the name of the pool is relevant here, so I changed this to an existing pool name. With git blame, I saw that when this file was created(https://github.com/apache/airflow/pull/1271), there was no `default_pool ` for tasks. 

Happy to hear your thoughts on this

Reply:
Anyone knows the context when #1225 was filed? Was it possible to schedule a task _without_ a pool back then?